Realm/RLMGeospatial.mm

```diff
@@ -0,0 +1,230 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2023 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMGeospatial.h"
+
+#import <realm/geospatial.hpp>
+
+@implementation RLMGeospatial
+- (realm::Geospatial)geoSpatial {
+    return realm::Geospatial{realm::GeoBox{realm::GeoPoint{1.1, 2.2}, realm::GeoPoint{1.1, 2,2}}};
```

Comment:
What do the values `{1.1, 2.2}` mean?

Reply:
Changed this to a protocol, so we don't need to override it and doesn't need a default value